### ISSUE TITLE
Use semantic versioning and releases

### ISSUE THREAD

**Issue by Saethox** [mahf-opt/mahf#132]
Now that we are public (or at least after we release the tech report), having versioning and releases would be appropriate. Especially if we have extension repos, which currently would instantly break the second we push API changes on `master`.

**Comment by HeleNoir** [comment]
Yes, we will definitely start versioning. I would like the first release to be the version for the Technical Report, and the same for the extension repos (or at least those we need and that will got public, too).
